### PR TITLE
Add tests for webui filters

### PR DESCRIPTION
**Pull request by skshetry**
This adds a tests for different filters in the activity.

### CODE REVIEW

File: tests/acceptance/features/webUIActivity/updateFilesFolder.feature [hunk 1]
```diff
@@ -15,14 +15,40 @@ Feature: Updated files/folders activities
...
     And the activity number 2 should contain message "You created text1.txt, text.txt" in the activity page
+    Examples:
+      | filter            |
+      | All Activities    |
+      | Activities by you |
+    # Favorites currently shows the creation. After fix, remove this.
+      | Favorites         |
+
+  @issue-622
+  Scenario Outline: Changing multiple file contents should not be shown in activity log for following filters
```
Comment (paurakhsharma) [inline tests/acceptance/features/webUIActivity/updateFilesFolder.feature]:
```suggestion
  Scenario Outline: Changing multiple file contents should not be shown in activity list for following filters
```

File: tests/acceptance/features/webUIActivity/updateFilesFolder.feature [hunk 2]
```diff
@@ -15,14 +15,40 @@ Feature: Updated files/folders activities
     Then the activity number 1 should have message "You changed text.txt" in the activity page
     And the activity number 2 should contain message "You created text.txt" in the activity page
 
-  Scenario: Changing multiple file contents should be shown in activity log
+  @issue-622
+  Scenario Outline: Changing multiple file contents should be shown in activity log for following filters
```
Comment (paurakhsharma) [inline tests/acceptance/features/webUIActivity/updateFilesFolder.feature]:
```suggestion
  Scenario Outline: Changing multiple file contents should be shown in activity list for following filters
```

File: tests/acceptance/features/webUIActivity/deleteAndRestore.feature [hunk 3]
```diff
@@ -66,14 +88,40 @@ Feature: Deleted and Restored files/folders activities
...
     And user "user0" has deleted file "textfile0.txt"
     And user "user0" has deleted folder "folder with space/simple-empty-folder"
     When the user browses to the activity page
+    And the user filters activity list by "<filter>"
     Then the activity number 1 should have message "You deleted simple-empty-folder, textfile0.txt, 'single'quotes, testavatar.png and 0" in the activity page
+    Examples:
+      | filter            |
+      | All Activities    |
+      | Activities by you |
+    # Favorites currently shows the creation. After fix, remove this.
```
Comment (paurakhsharma) [inline tests/acceptance/features/webUIActivity/deleteAndRestore.feature]:
```suggestion
    # Favorites currently shows the deletion. After fix, remove this.
```

File: tests/acceptance/features/webUIActivity/deleteAndRestore.feature [hunk 4]
```diff
@@ -8,10 +8,32 @@ Feature: Deleted and Restored files/folders activities
...
+  Scenario Outline: file deletion should be listed in the activity list for following filters
     Given user "user0" has deleted file "lorem.txt"
     When the user browses to the activity page
+    And the user filters activity list by "<filter>"
     Then the activity number 1 should have message "You deleted lorem.txt" in the activity page
+    Examples:
+      | filter            |
+      | All Activities    |
+      | Activities by you |
+    # Favorites currently shows the creation. After fix, remove this.
```
Comment (paurakhsharma) [inline tests/acceptance/features/webUIActivity/deleteAndRestore.feature]:
```suggestion
    # Favorites currently shows the deletion. After fix, remove this.
```

File: tests/acceptance/features/webUIActivity/createFilesFolder.feature [hunk 5]
```diff
@@ -88,9 +180,33 @@ Feature: Created files/folders activities
...
+    Then the activity number 1 should contain message "You created text.txt, text.txt, doc" in the activity page
+    Examples:
+      | filter            |
+      | All Activities    |
+      | Activities by you |
+    # Favorites currently shows the creation. After fix, remove this.
+      | Favorites         |
+
+  @issue-622
+  Scenario Outline: Copying files to another folder should not be shown in log for following filters
```
Comment (paurakhsharma) [inline tests/acceptance/features/webUIActivity/createFilesFolder.feature]:
```suggestion
  Scenario Outline: Copying files to another folder should not be shown in activity list for following filters
```

File: tests/acceptance/features/webUIActivity/createFilesFolder.feature [hunk 6]
```diff
@@ -88,9 +180,33 @@ Feature: Created files/folders activities
     When the user browses to the activity page
     And the activity number 1 should contain message "You created doc2, doc" in the activity page
 
-  Scenario: Copying files to another folder should be shown in log
+  @issue-622
+  Scenario Outline: Copying files to another folder should be shown in log for following filters
```
Comment (paurakhsharma) [inline tests/acceptance/features/webUIActivity/createFilesFolder.feature]:
```suggestion
  Scenario Outline: Copying files to another folder should be shown in activity list for following filters
```

File: tests/acceptance/features/webUIActivity/createFilesFolder.feature [hunk 7]
```diff
@@ -8,22 +8,90 @@ Feature: Created files/folders activities
...
 
-  Scenario: Creating new file should be listed in the activity list
+  @issue-622
+  Scenario Outline: Creating new file should be listed in the activity list
     Given user "user0" has uploaded file "filesForUpload/textfile.txt" to "/text.txt"
     When the user browses to the activity page
+    And the user filters activity list by "<filter>"
     Then the activity number 1 should contain message "You created text.txt" in the activity page
+    Examples:
+    | filter            |
```
Comment (individual-it) [inline tests/acceptance/features/webUIActivity/createFilesFolder.feature]: indent

**Pull request merged**